src/client/Microsoft.Identity.Client/AppConfig/ITelemetryEventPayload.cs

```diff
@@ -1,14 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 
 namespace Microsoft.Identity.Client
 {
     /// <summary>
     /// Data that represents a single snapshot in the series of events that are collected
     /// </summary>
     /// <remarks>This API is experimental and it may change in future versions of the library without an major version increment</remarks>
+    [Obsolete("Telemetry is sent automatically by MSAL.NET. See https://aka.ms/msal-net-telemetry.", false)]
```

Comment:
>false [](http://example.com/codeflow?start=101&length=5)

Should we obsolete this with error?

Reply:
Even though no one should be using this, it's still a breaking change? I think we only obsoleted with error for experimental APIs.